src/contract/userop/contract.ts

```diff
@@ -0,0 +1,141 @@
+
+import { ethers, utils, ContractInterface, Signer } from "ethers";
+import { UserOperationEventEvent } from "userop/dist/typechain/EntryPoint";
+import { VerifiedWallet } from "../../wallet";
+import { Client, Presets, UserOperationMiddlewareFn } from "userop";
+
+export enum USEROP_DATATYPES {
+    VALUE = 'number',
+    STRING = 'string',
+    ADDRESS = 'address',
+}
+
+const response: { transactionHash: string | null, result: UserOperationEventEvent | null } = { transactionHash: null, result: null }
+export type UseropResponse = typeof response;
+
+
+export class UseropVerifiedContract {
+    private paymasterUrl:string
+    private apiKey:string
+    private walletAddress:string
+    private bundlerUrl:string
+    private useropSigner: ethers.Wallet;
+    private paymasterMiddleware: UserOperationMiddlewareFn
+
+    constructor(signer: VerifiedWallet, apiKey:string, paymasterUrl:string, wallet:VerifiedWallet ) {
```

Comment:
@herberthk 

This change in constructor will break all constructors of the sdk contracts.